### ISSUE TITLE
After Installing YTR Modules, My Data Got Encrypted

### ISSUE THREAD

**Issue by Attached2Android** [j-hc/revanced-magisk-module#77]
Yesterday, I Installed Youtube Revanced Module In My Device, And When I Reboot My Device, Then My Device Is In Bootloop, So I Reboot To Recovery Mode, Then I See My DATA Got Encrypted, Im Not Able To Fix It, I Have To FORMAT The Data To Bring Device In Working Mode.

Device : POCO M2 PRO
ROM - MIUI 13 India
Android Version - 12

**Comment by j-hc** [comment]
pretty unrelated

**Comment by Attached2Android** [comment]
I Also Think It's Not Related To This Module, But Today I Installed It Again, After That I Reboot, Then Device In Bootloop, So Reboot To Recovery, Then I Saw My Data Encryted And I'm Not Able To Mount System Folder.

I Don't Know Why It Happed.

I Debloat YouTube Using "De-Bloater" App, May Be Due To This, It's Happened.

**Comment by j-hc** [comment]
all this module does is installing the youtube as a user app and bind mounting revanced on top of ir. it cant encrypt your data partition. if you have other modules or having system crashes or anything that could cause unmount/remount of the/data can be problematic. there isnt much I can do.

**Comment by Attached2Android** [comment]
Yes, I'm Also Think That. BTW Thanks For Replying. But Now I Don't Have Courage To Try Again, May If That Is Not The Cause Then It's Very Terrible Situation For Me.

BTW Appreciate Your Work. Keep It Up Sir.
